tests/graph/test_op.py

```diff
@@ -232,3 +232,55 @@ def perform(self, *_):
 
     x = pt.TensorType(dtype="float64", shape=(1,))("x")
     assert SomeOp()(x).type == pt.dvector
+
+
+def test_op_name():
+    op_name = "op_name"
+
+    class DummyType(Type):
+        def filter(self, data):
+            return data
+
+        def __eq__(self, other):
+            return isinstance(other, DummyType)
+
+        def __hash__(self):
+            return hash(DummyType)
+
+        def __repr__(self):
+            return "DummyType()"
+
+    def dummy_variable(name):
+        return Variable(DummyType(), None, None, name=name)
+
+    x = dummy_variable("x")
+
+    class SingleOutOp(Op):
+        def make_node(self, *inputs):
+            outputs = [dummy_variable("a")]
+            return Apply(self, list(inputs), outputs)
```

Comment:
Traditionally, an Op would not assign a default name in `make_node`
  
```suggestion
        def make_node(self, input):
            inputs = [input]
            outputs = [input.type()]
            return Apply(self, inputs, outputs)
```